src/partners/certify-hardware.njk

```diff
@@ -0,0 +1,93 @@
+    </div> 
+</div>
+<div class="mt-8 md:mt-10">
+    {% include "components/divider-flow--top-red--light.njk" %}
+</div>
+<div class="ff-bg-light w-full px-6 pt-10 pb-20 md:px-0 flex flex-col gap-10 md:gap-12">
+    <div class="container max-w-screen-md m-auto flex flex-col gap-4">
+        <h3 class="m-auto text-center w-full mb-6 max-w-xs sm:max-w-none">Technical Qualification in 3 Steps</h3>
+        <div class="ff-certify-list">
+            <div class="ff-item">
+                <div class="ff-line-container sm:hidden">
+                    <div class="ff-line"></div> 
+                </div>  
+                <div class="ff-certify-list-icon sm:pb-2">{% include "components/icons/book-open.svg" %}</div>
+                <div class="ff-line-container">
+                    <div class="ff-line"></div> 
+                </div>                
+                <div class="sm:row-span-2 col-span-3 sm:col-auto pt-6 sm:pt-0">Explore the FlowFuse Device Agent <a href="https://github.com/FlowFuse/device-agent" target="_blank">Repository</a> and <a href="/../docs/device-agent/introduction/">Documentation</a>.</div>
+            </div>
+            <div class="ff-item">
+                <div class="ff-line-container">
+                    <div class="ff-line"></div> 
+                </div>  
+                <div class="ff-certify-list-icon">{% include "components/icons/check-circle.svg" %}</div>
+                <div class="ff-line-container">
+                    <div class="ff-line"></div> 
+                </div>                
+                <div class="sm:row-span-3 col-span-3 my-auto pt-6 sm:pt-0">Ensure that the Device Agent comes pre-installed on your device. Alternatively, it should be available from a store or marketplace as a "one-click" installer.</div>
+            </div>
+            <div class="ff-item">
+                <div class="ff-line-container">
+                    <div class="ff-line"></div> 
+                </div>  
+                <div class="ff-certify-list-icon">{% include "components/icons/document-arrow-up.svg" %}</div>
+                <div class="ff-line-container">
+                    <div class="ff-line sm:hidden"></div> 
+                </div>                
+                <div class="sm:row-span-3 col-span-3 my-auto pt-6 sm:pt-0">Users can easily upload their device configuration either through a custom user interface (e.g., file upload or input mask) or via the built-in Device Agent UI.</div>
+            </div>
+        </div>        
+        {% include "components/divider-grey-wires-right.njk" %}
+    </div>
+    <div class="container max-w-screen-md m-auto flex flex-col gap-4">
+        <h3 class="m-auto text-center w-full">Certification</h3>
+        <div class="flex flex-col md:flex-row md:gap-10 gap-6 md:pb-6 md:px-10">
+            <p class="md:w-[65%] w-full">Lorem ipsum dolor sit amet consectetur. A turpis amet ullamcorper volutpat tellus ut sagittis sit.</p>
```

Comment:
@MarianRaphael this is the placeholder text we talked about

Reply:
```suggestion
            <p class="md:w-[65%] w-full">After successfully navigating the technical qualification, take the next step by booking a meeting with our team. Together, we'll work towards ensuring your device is fully compatible with FlowFuse and Node-RED</p>
```